docs/server/source/production-deployment-template-tendermint/architecture.rst

```diff
@@ -0,0 +1,204 @@
+Architecture of a BigchainDB Node
+==================================
+
+A BigchainDB Production deployment is hosted on a Kubernetes cluster and includes:
+
+* NGINX, OpenResty, BigchainDB, MongoDB and Tendermint
+  `Kubernetes Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+* NGINX, OpenResty, BigchainDB, Monitoring Agent and Backup Agent
```

Comment:
1. "Monitoring Agent and Backup Agent" are mentioned here but elsewhere only Monitoring Agent seems to be mentioned.
2. I'd write "MongoDB Monitoring Agent" just to be more clear (i.e. add "MongoDB" in front).
3. Neither agent seems to be on the diagram.

Reply:
Done.

Reply:
:+1: